### ISSUE TITLE
Can't see inline diffs using the `Night Wolf` theme.

### ISSUE THREAD

**Issue by Avivhdr** [mao-santaella-rs/NightWolfTheme#33]
Hi @mao-santaella-rs,
Great work on this theme 😍

In the below gif, please notice the text `.asset` in line 22.
Using the `Night Owl` variations it is possible to see the changes while using `Night Wolf` variations it is not possible.

![wolf](https://user-images.githubusercontent.com/23265867/143858310-dbcca359-2909-49f4-bb1d-dbd48ddaad3a.gif)
)

**Comment by mao-santaella-rs** [comment]
Hi @Avivhdr thanks for taking the time to do this issue, I don’t really know what the problem would be in this case, maybe should be that this theme doesn't really do many customizations on the .tsx files if you can provide me with a .tsx file that has all or as much as you can with examples of code I maybe can, I can check if I can fix the issue.
Thanks again

**Comment by Avivhdr** [comment]
Hi @mao-santaella-rs 
The problem is not in the type of file. It happens in all files.

Steps to reproduce:

1. create a new file with this name (`RelayDeferredModal.test.tsx` - the name doesn't really matter) and with this content:
```
import React, { Component } from 'react'
import PropTypes from 'prop-types'

export default class aaa-before-aaa extends Component {
  static propTypes = {
    prop: PropTypes
  }

  render() {
    return (
      <div>
        Hello
      </div>
    )
  }
}
```

2. Stage changed with `git add .`.

3. go to the file and change the word `before` to `after`.

4. open the `source-control` tab on vscode.

5. You will see the file twice, click on the lower one:
![image](https://user-images.githubusercontent.com/23265867/148113043-cc14eb54-215f-4671-a8f7-21eecf327c62.png)

you will see this:
![image](https://user-images.githubusercontent.com/23265867/148113165-abaab178-c1ba-4fc4-8978-2f9ea949136b.png)

6. Change the theme to `Dark (default dark)`, and you will see:
![image](https://user-images.githubusercontent.com/23265867/148113403-00363619-18b7-479d-a797-092b18304a69.png)

  Do you see the difference between them in the words before & after?

**Comment by chalop** [comment]
To add to the conversation, I think the issue itself is how Night Wolf theme handles highlighting (notice the `.asset` in the original post/gif).

This might be related, but when I highlight text on any input box the highlight does not appear.

**Comment by mao-santaella-rs** [comment]
Hi @chalop, I had tried to solve this issue and for the life of me idk how to fix it, do you think you can give it a try? I don't mind adding it to the theme if you fix this.

**Comment by Avivhdr** [comment]
I solved it locally by adding these settings to my `settings.json`:
```
    "diffEditor.insertedTextBackground": "#99b76d23",
    "diffEditor.insertedTextBorder": "#c5e47833",
    "diffEditor.removedTextBackground": "#ef535033",
    "diffEditor.removedTextBorder": "#ef53504d",
```

* colors were copied from the night wolf theme:
https://github.com/sdras/night-owl-vscode-theme/blob/main/themes/Night%20Owl-color-theme-noitalic.json#L115-L118

Not ideal colors, but better than how it is now.

**Comment by mao-santaella-rs** [comment]
wow nice man I'll add this as soon as I can, thanks a lot!

**Comment by mao-santaella-rs** [comment]
hi @Avivhdr, I fixed this in the latest version (1.12.0), let me know if you need anything else.

thanks again and sorry for the delay
